Makefile

```diff
@@ -1,4 +1,5 @@
-all    :; dapp build
-clean  :; dapp clean
-test   :; dapp test --rpc-url ${ETH_RPC_URL}
-deploy :; dapp create Convex
+benchmark-all       :; bash scripts/benchmark-all.sh
```

Comment:
Aside: does this repo need to do any results verification? One or more tool's simulation could be totally inaccurate and we wouldn't know!

Reply:
Yep true! I was thinking of also reporting the `gasUsed` of each tool in a future PR, but open to other ideas here also. Not sure if verifying state changes or comparing transaction traces is excessive for a benchmarking repo 🤔

Reply:
I was just thinking... if we're forking at the same block for each tool, just count the logs and make sure all tools report the same number of logs. Seems quick and dirty? Could also do a deep-equals on the JSON, but that's perhaps more prone to minute differences.

Anyway, this also might be something to manually test once and then not worry about until/unless this repo becomes used more seriously.

Reply:
True, just verifying once might be sufficient for now! Either way, now tracking all potential metrics in https://github.com/mds1/convex-shutdown-simulation/issues/3